### DISCUSSION TITLE
Welcome to discussions on spring-boot-backend!

### DISCUSSION THREAD

**Discussion by bglamadrid** [trebol-ecommerce/trebol-backend-monolith#153] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

**Comment by mepox** [comment]
First 😅
